python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -392,7 +397,7 @@ class NearestNeighbors(Base):
 
         self.n_dims = n_dims
 
-    def kneighbors(self, X, k=None, convert_dtype=False):
+    def kneighbors(self, X, k=None, convert_dtype=True):
```

Comment:
Do we want to set convert_dtype here if _should_downcast == true?

Reply:
i believe so, because if the user is using `should_downcast` variable then we want to set `convet_dtypes` to True so we can convert from float64 to float32

Reply:
Oops. I missed that it was doing this in. That's what I get for only looking at the diff!